Testscripts/Linux/validate-intel-sgx-driver.sh

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env bash
+#######################################################################
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache License.
+#
+# validate-intel-sgx-driver.sh
+# Description:
+#    Check the existence of Intel sgx driver.
+#    Install Open-Enclave and build&run Samples
+# Supported Distros:
+#    Ubuntu
+#######################################################################
+
+set -e
+set -x
+
+. utils.sh || {
+    echo "Error: unable to source utils.sh!"
+    echo "TestAborted" >state.txt
+    exit 1
+}
+
+UtilsInit
+
+sudo apt-get update
+sudo apt-get -y upgrade
+echo 'deb [arch=amd64] https://download.01.org/intel-sgx/sgx_repo/ubuntu bionic main' | sudo tee /etc/apt/sources.list.d/intel-sgx.list
+wget -qO - https://download.01.org/intel-sgx/sgx_repo/ubuntu/intel-sgx-deb.key | sudo apt-key add -
+
+echo "deb http://apt.llvm.org/bionic/ llvm-toolchain-bionic-7 main" | sudo tee /etc/apt/sources.list.d/llvm-toolchain-bionic-7.list
+wget -qO - https://apt.llvm.org/llvm-snapshot.gpg.key | sudo apt-key add -
+
+echo "deb [arch=amd64] https://packages.microsoft.com/ubuntu/18.04/prod bionic main" | sudo tee /etc/apt/sources.list.d/msprod.list
+wget -qO - https://packages.microsoft.com/keys/microsoft.asc | sudo apt-key add -
+sudo apt-get update
+
+LogMsg "----- Checking sgx driver -----"
+if ! modinfo intel_sgx; then
+    LogErr "modinfo intel_sgx failed"
+    SetTestStateFailed
+    exit 1
+fi
+if [ "$(stat -c %a /dev/sgx)" == 600 ]; then
+    LogMsg "/dev/sgx permission is incorrect: 0600, temporarily fix this issue by chmod 666"
+    sudo chmod 666 /dev/sgx
+fi
+
+LogMsg "----- Install Open Enclave packages and dependencies -----"
+sudo apt -y install clang-7 libssl-dev gdb libsgx-enclave-common libsgx-enclave-common-dev libprotobuf10 libsgx-dcap-ql libsgx-dcap-ql-dev az-dcap-client open-enclave
```

Comment:
There is a common function call 'install_package' from utils.sh. This is how we handle the same package installation in different distro. Suggest to use this function call instead of Ubuntu specific command.

Reply:
right now the intel driver is only ubuntu. we do not have the supporting software from Intel yet for other distros

Reply:
There is a bug in install_package such that it doesn't support multiple packages.

Reply:
use double quotes install_package "git wget"

Reply:
I don't see your change or commits after then, but you close this comment. What does it mean?

Reply:
@juhlee-microsoft I resolved this comment right after my last comment. @LiliDeng I saw this https://github.com/LIS/LISAv2/blob/master/Testscripts/Linux/utils.sh#L2036 and thought it was a bug: package_name=$1 should be package_name="$1". But despite my suspicion, install_package "git wget" actually works.